container_registry/container_analysis/poll_discovery_finished.go

```diff
@@ -52,8 +52,19 @@ func pollDiscoveryOccurrenceFinished(resourceURL, projectID string, timeout time
 		case <-ticker.C:
 			req := &grafeaspb.ListOccurrencesRequest{
 				Parent: fmt.Sprintf("projects/%s", projectID),
-				Filter: fmt.Sprintf(`kind="DISCOVERY" AND resourceUrl=%q`, resourceURL),
+				// Vulbnerability discovery occurrences are always associated with the
+				// PACKAGE_VULNERABILITY note in the "goog-analysis" GCP project.
+				Filter: fmt.Sprintf("resourceUrl=\"%s\" AND noteProjectId=\"%s\" AND noteId=\"%s\"",
+					resourceUrl, "goog-analysis", "PACKAGE_VULNERABILITY"),
```

Comment:
No need to use placeholders for static strings? Also, can use `%q` to automatically put quotes around the value. Can also use backticks (`) to avoid having to escape ", if you need.

Reply:
Yeah, I thought bringing the static strings out made it easier to read, both by displaying the template format more clearly and making the line shorter. I can change it if you think it's better to include them in the string though, I don't feel too strongly about it

i didn't know about %q, I'll add that